package.json

```diff
@@ -32,6 +32,9 @@
     "slugify": "^1.6.5"
   },
   "devDependencies": {
+    "@typescript-eslint/parser": "^5.47.0",
+    "eslint": "^8.30.0",
+    "eslint-plugin-notice": "^0.9.10",
```

Comment:
That sure is a lot of dependencies, but if it works, alright.

Reply:
yeah, I am also not super happy about it